config.example.yaml

```
@@ -5,5 +5,10 @@ permissions:
 jwt:
   jwksuri: https://identity-api.enterprise.dev/jwks.json
   issuer: https://identity-api.enterprise.dev/
+events:
+  nats:
+    url: localhost:4222
```

Does this need the protocol?
```suggestion
    url: nats://localhost:4222
```

---

Probably good to add it regardless.